polylogue/cli/arg_helpers.py

```
@@ -34,6 +39,212 @@ def add_force_option(parser: ArgumentParser, *, help_text: str) -> None:
+            path.mkdir(parents=True, exist_ok=True)
+            return path
+        except OSError as exc:
+            if json_mode:
+                from .json_output import JSONModeError
+
+                raise JSONModeError(
+                    "path_create_failed",
+                    f"Could not create directory: {path}",
+                    path=str(path),
+                    reason=str(exc),
+                )
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)